ci/Debian11Container.dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:lts-bullseye
+WORKDIR /
+SHELL ["/bin/bash", "-c"]
+
+# Install emscripten
+RUN git clone https://github.com/emscripten-core/emsdk.git \
+    && emsdk/emsdk install 3.1.59 \
+    && emsdk/emsdk activate 3.1.59 \
+    && source emsdk/emsdk_env.sh
+ENV PATH="$PATH:/emsdk/upstream/bin:/emsdk/upstream/emscripten"
+
+# Install FPM
+RUN apt-get update && apt-get install -y ruby && gem install fpm:1.15.1
+
+# Install pkcs11-tool from OpenSC
```

Comment:
to sign desktop clients and deb packages. We should  also mention that we need `osslsigncode` for windows app specifically

Reply:
See: https://github.com/tutao/tutanota/pull/7297#discussion_r1695409559.